C++/syntax_test_c.c

```diff
@@ -437,8 +437,16 @@ if (1) {
 if (2) {
 # elif BAZ
 if (3) {
-# else
+#elifdef BAR2
```

Comment:
What Michael is asking for is adding some test assertions.

Example

```suggestion
#elifdef BAR2
/* <- keyword.control.import */
/*^^^^^^ keyword.control.import */
```

The choosen section however seems to rather be for testing correct handling of multiple `{` with intermediate directives.

Tests targeting `#ifdef` seem to be located at https://github.com/sublimehq/Packages/blob/493cb8218a2d77128fb54ebf096256aa5cdeb019/C%2B%2B/syntax_test_c.c#L583-L584